website/addons/wiki/routes.py

```diff
@@ -112,17 +120,14 @@
             '/project/<pid>/node/<nid>/wiki/<wname>/rename/',
         ], 'put', views.project_wiki_rename, json_renderer),
 
-        # Compare : GET
-        # <wver> refers to a wiki page's version number
-        Rule([
-            '/project/<pid>/wiki/<wname>/compare/<int:wver>/',
-            '/project/<pid>/node/<nid>/wiki/<wname>/compare/<int:wver>/',
-        ], 'get', views.project_wiki_compare, json_renderer),
-
         # Delete : DELETE
         Rule([
             '/project/<pid>/wiki/<wname>/',
             '/project/<pid>/node/<nid>/wiki/<wname>/',
+            '/project/<pid>/wiki/<wname>/edit/',
```

Comment:
This change was made in regards to a intermediary routing system used in development. I will remove these.